### DISCUSSION TITLE
How is a non-functional eye-gaze situation typically mitigated?

### DISCUSSION THREAD

**Discussion by jakemcivor** [ATMakersOrg/EyeAlert#6] [Q&A]
1. It's not.
2. Regular checks by a caregiver.
3. ?

**Comment by ImproveAbility** [comment]
- user's ventilator alarms because they need suction and were not able to get caregiver's attention before it got to a bad situation.

**Comment by NachumLehman** [comment]
Everything depends upon what stage of ALS we're discussing.  If we assume we're talking about someone who can ONLY move their eyes, the only possible alert would be when a caregiver sees them signaling them with their eyes or crying out of desperation.

**Comment by JayToltTech** [comment]
Grunts.  Face turns red due to frustration/anger.

**Reply by JayToltTech** [reply]
Really, a machine learning classifier that can detect flushed facial capillaries due to frustration or distress would be a really good early indicator, at least based on my own limited experience.
